voila/app.py

```diff
@@ -133,52 +133,16 @@ def parse_command_line(self, argv=None):
         self.template_paths = []
         self.static_paths = [self.static_root]
         if self.template:
-            self._collect_template_paths(self.template)
+            collect_template_paths(
+                self.nbconvert_template_paths,
```

Comment:
`nbconvert_template_paths` is already the list of paths, no need to search again. However, I think we **should** move this path searching to the handler, so we can have per notebook or parametrized templates.